### PR TITLE
Add the labels flowchart to the README.

### PR DESCRIPTION
**Pull request by ezio-melotti**
This PR adds a flowchart that describe the PR state machine to the `README`.  

Related issues:
* The flowchart will be linked in https://github.com/python/devguide/pull/930
* The idea was brought up in https://github.com/python/devguide/issues/868

The flowchart is described in
https://github.com/python/bedevere/blob/428290e46a9507c7810614e15294163d2b4f9eb0/bedevere/stage.py#L3-L38
which renders to

image

For the implementation I used mermaid, which is supported natively by GitHub-flavored Markdown.  The new flowchart renders to:
```mermaid
flowchart TD
    A([New PR]):::creator
    A -- by contributor --> B[Awaiting review]:::anyone
    A -- by core dev --> C[Awaiting core review]:::coredev
    B & C -- new review by\nanother contributor --> C
    C & B & E  -- new core review\nrequests changes --> D[Awaiting changes]:::creator
    D -- changes by contributor --> E[Awaiting change review]:::coredev
    C & E & B -- new core review\napproves ---> F[Awaiting merge]:::coredev
classDef creator stroke:#CC0;
classDef anyone stroke:#00C;
classDef coredev stroke:#0C0;
linkStyle 0,1,7 stroke:#CC0,color:auto;
linkStyle 2,3 stroke:#00C,color:auto;
linkStyle 4,5,6,8,9,10 stroke:#0C0,color:auto;
```
<details>
<summary>Click to see a screenshot of the flowchart (using the dark theme)</summary>

image
</details>

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Should we remove the digraph from comments in `stage.py` and refer to this one?

**Comment by DanielNoord** [comment]
Could this also close https://github.com/python/bedevere/issues/184?

**Comment by ezio-melotti** [comment]
> Should we remove the digraph from comments in stage.py and refer to this one?

I was thinking about that :thinking:
In the new graph I tried to include everything that was included in the old one, with the exception of the CSS class for triagers (since it's unused).  I don't think we are losing any information by deleting it, and I can always leave a comment mentioning it in case someone needs it.

**Comment by ezio-melotti** [comment]
I removed the old graph and added a couple of comments.

File: bedevere/stage.py [hunk 1]
```diff
@@ -1,41 +1,11 @@
...
-
-  "New PR" -> "Awaiting core review" [label="New PR by core devs", color=green]
-}
-"""
+# The state machine for the flow of a PR is currently
+# documented with a Mermaid graph in the README.
+# The graph replaces a previous version (written as
+# a DOT file) that was included here.
+#
+# Changes to this file should be reflected in the README.
```
Comment (DanielNoord) [inline bedevere/stage.py:8]: Is this line necessary?

I'm not even sure the comments are necessary. They are probably relevant now but in 6/12 months time they will probably just become dead weight. I would be okay with removing them as well, but I'll leave that decision to you!

**Review by DanielNoord — APPROVED** [review]

File: bedevere/stage.py (hunk 1 above)
Comment (ezio-melotti) [reply bedevere/stage.py:8]: The one about updating the README is important to ensure the graph in the README keeps matching the code (and in turn the comment in the README ensures the devguide is updated -- assuming people see the comments).

The one about the previous version is not particularly important, but without it someone might end up wasting time looking for that graph they once saw in the code.  Having a couple of extra lines for this comment is not a big cost.

**Pull request merged**